app/helpers/audit_helper.rb

```diff
@@ -112,7 +112,9 @@ def activity(type_of_activity, args = nil)
   end
 
   def audit_user_name(audit)
-    if audit.api_user.present?
+    if applicant_activity_types.include?(audit.activity_type)
+      t("audit_user_name.bops_applicants")
+    elsif audit.api_user.present?
```

Comment:
This whole file is kind of wild with all the if/else statements. Feels like there's maybe a better way to be doing it, not that I've thought about it that much.

Reply:
There definitely is but just wanted to get a fix out without spending too much time on this